pkg/kubelet/kubelet.go

```diff
@@ -169,9 +169,9 @@ const (
 	genericPlegRelistThreshold = time.Minute * 3
 
 	// Generic PLEG relist period and threshold when used with Evented PLEG.
-	eventedPlegRelistPeriod     = time.Second * 300
-	eventedPlegRelistThreshold  = time.Minute * 10
-	eventedPlegMaxStreamRetries = 5
+	genericPlegRelistPeriodWhenEventedPlegInUse    = time.Second * 300
+	genericPlegRelistThresholdWhenEventedPlegInUse = time.Minute * 10
```

Comment:
Can you point out where this renaming was requested?

Reply:
https://github.com/kubernetes/kubernetes/pull/111384#discussion_r1016848242